src/cryptography/hazmat/primitives/serialization/pkcs7.py

```diff
@@ -232,6 +264,13 @@ def encrypt(
             raise ValueError("Must have at least one recipient")
         if self._data is None:
             raise ValueError("You must add data to encrypt")
+
+        # The default content encryption algorithm is AES-128, which the S/MIME
+        # v3.2 RFC specifies as MUST support (https://datatracker.ietf.org/doc/html/rfc5751#section-2.7)
+        self._content_encryption_algorithm = (
+            self._content_encryption_algorithm or algorithms.AES128
+        )
```

Comment:
Don't mutate self here, just use a local. You'll get wrong behavior on a reused builder.

Reply:
Good catch! I'll need to modify the rust code accordingly

Reply:
Done!